### PR TITLE
Fix linking of libcvmfs_crypto

### PR DESCRIPTION
**Pull request by jblomer**
Trying to understand the libcvmfs S3 error

### CODE REVIEW

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

File: cvmfs/crypto/CMakeLists.txt [hunk 1]
```diff
@@ -13,6 +13,8 @@ set (LIBCVMFS_CRYPTO_SOURCES
 )
 
 set (LIBCVMFS_CRYPTO_CFLAGS "-D_FILE_OFFSET_BITS=64 -DCVMFS_LIBRARY -DCVMFS_RAISE_EXCEPTIONS -fexceptions")
+# Prevent symbold from static crypto libs to be re-exported
```
Comment (vvolkl) [inline cvmfs/crypto/CMakeLists.txt]:
```suggestion
# Prevent symbols from static crypto libs to be re-exported
```
typo

**Comment by vvolkl** [comment]
Nice!

**Review by vvolkl — APPROVED** [review]

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Pull request merged**